src/main/javascript/package.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "ccif-js-model-generator",
+  "name": "cif-js-model-generator",
   "version": "0.1.0",
-  "description": "Generates ES6 model classes based on the CCIF Swagger definitions.",
+  "description": "Generates ES6 model classes, Nginx configuration, and graphQL schema based on the CIF Swagger specification.",
   "main": "index.js",
   "private": true,
   "scripts": {
     "test": "mocha test/**/*Test.js",
-    "build": "node index.js; node nginx.js; npm run graphql; npm run prepareRelease",
+    "build": "npm run models; npm run nginx; npm run graphql; npm run prepareRelease",
```

Comment:
Not a show stopper, but maybe we can invest some time looking into a "task runner" framework so that we don't have to `npm this` and `npm that` all the time...